code/modules/client/preference_setup/loadout/items/factions.dm

```diff
@@ -351,13 +351,62 @@
 	pmcg_headwear["Grupo Amapola utility cover"] = /obj/item/clothing/head/grupo_amapola/ute
 	gear_tweaks += new /datum/gear_tweak/path(pmcg_headwear)
 
+// START: PMCG MODSUIT
 /datum/gear/faction/pmc_modsuit
 	display_name = "PMCG modsuit"
 	description = "A modular PMCG fatigue jumpsuit."
 	path = /obj/item/clothing/under/pmc_modsuit
 	slot = slot_w_uniform
 	faction = "Private Military Contracting Group"
 
+/datum/gear/faction/pmc_modsuit/New()
+	..()
+	gear_tweaks += list(gear_tweak_modsuit_configuration)
+
+
+var/datum/gear_tweak/modsuit_configuration/gear_tweak_modsuit_configuration = new()
+
+/datum/gear_tweak/modsuit_configuration
+	var/list/configuration_options = list()
+
+/datum/gear_tweak/modsuit_configuration/New()
+	..()
+
+	var/obj/item/clothing/under/pmc_modsuit/initial_uniform = new()
```

Comment:
I don't think we need to instantiate this? Initial / :: should be sufficient

Reply:
iirc, initial doesn't work on lists, it has to be initialized and deleted